### PR TITLE
Chore/bump nexus 2.6.9

### PR DESCRIPTION
**Pull request by falmesino**
```
Chore/bump nexus 2.6.9
- bump nexus version from 2.5.3 to 2.6.9
- add shipping locales
- style input qty
- style discount note
- add and style buyer note input for each item in CartDetails and OrderSummaryBox
```

URL PR: https://github.com/sirclo-solution/template-rose/pull/12
URL Task: https://phabricator.sirclo.com/T33595
Pull Latest: OK
Build (Local): Failed, tapi sepertinya di harbormaster akan aman

Slack Thread URL: https://sirclo.slack.com/archives/C03HEEV3UDC/p1655804282496529

***

**Screenshots (item note title if not empty)**
Screen Shot 2022-06-22 at 17 35 43
Screen Shot 2022-06-22 at 17 42 43
Screen Shot 2022-06-22 at 17 35 18
Screen Shot 2022-06-22 at 17 42 57

**Screenshots (CartDetails) **

Screen Shot 2022-06-21 at 16 27 29
Screen Shot 2022-06-21 at 16 27 34
Screen Shot 2022-06-21 at 16 27 36
Screen Shot 2022-06-21 at 16 27 39

**Screenshots (OrderSummaryBox) - Baru**

Screen Shot 2022-06-22 at 16 27 49
Screen Shot 2022-06-22 at 16 27 52
Screen Shot 2022-06-22 at 16 27 57
Screen Shot 2022-06-22 at 16 28 01

**Screenshots (OrderSummaryBox) - Lama**

Screen Shot 2022-06-21 at 16 26 28
Screen Shot 2022-06-21 at 16 26 32
Screen Shot 2022-06-21 at 16 26 37
Screen Shot 2022-06-21 at 16 26 40

### CODE REVIEW

**Review by rhmnaulia — APPROVED** [review]

**Review by virginiaritonga — APPROVED** [review]

File: components/OrderSummaryBox/index.tsx [hunk 1]
```diff
@@ -69,21 +72,48 @@ const classesOrderSummary = {
...
+  itemQtyClassName: 'itemQty',
+	itemNoteClassName: styleOrderSummary.cart_itemNote,
   errorClassName: 'd-none',
   itemEditClassName: 'd-none',
-  itemRemoveClassName: 'd-none',
+  itemRemoveClassName: 'itemRemove',
   cartFooterClassName: 'd-none',
   cartFooterTitleClassName: 'd-none',
 }
+*/
```
Comment (inezamandha) [inline components/OrderSummaryBox/index.tsx:92]: ini diperluin gak ya mang? soalnya di comment

File: components/OrderSummaryBox/index.tsx [hunk 2]
```diff
@@ -130,9 +160,12 @@ const OrderSummaryComponent: FC<iProps> = ({
           <CartDetails
             currency="IDR"
             classes={classesCartDetails}
+						variantSeparator=", "
+						withSeparatedVariant
```
Comment (inezamandha) [inline components/OrderSummaryBox/index.tsx:164]: ini ke tab ya mang?

File: components/OrderSummaryBox/index.tsx [hunk 3]
```diff
@@ -130,9 +160,12 @@ const OrderSummaryComponent: FC<iProps> = ({
...
             itemRedirectPathPrefix={`product`}
+            isEditable={true}
+						removeIcon={<IoTrashBinOutline />}
```
Comment (inezamandha) [inline components/OrderSummaryBox/index.tsx:167]: ini juga yaah?

File: components/OrderSummaryBox/index.tsx [hunk 4]
```diff
@@ -142,6 +175,17 @@ const OrderSummaryComponent: FC<iProps> = ({
...
+            withProductNote
+            productNoteButtonElement={{
+						  filled: <span>{i18n.t("cart.change")}</span>,
+						  save: <span>{i18n.t("cart.save")}</span>,
+						  empty: (
+							<>
+                <FaEdit />
+							  <span>{i18n.t("cart.addNote")}</span>
+							</>
+						  )
```
Comment (inezamandha) [inline components/OrderSummaryBox/index.tsx:187]: ini ke tab apa emang begini yaa mang?

**Review by inezamandha — CHANGES_REQUESTED** [review]

File: components/OrderSummaryBox/index.tsx (hunk 4 above)
Comment (falmesino) [reply components/OrderSummaryBox/index.tsx]: sudah dirapiken, abis ngurutin props-nya biar sama kaya yang di cart

File: components/OrderSummaryBox/index.tsx (hunk 1 above)
Comment (falmesino) [reply components/OrderSummaryBox/index.tsx]: sudah dihapus

**Review by inezamandha — APPROVED** [review]

**Review by rhmnaulia — APPROVED** [review]

**Review by mayun19 — CHANGES_REQUESTED** [review]
mang otoy, di figma ada label "Catatan untuk penjual". ini belum ditambahan kah(?)

**Review by mayun19 — APPROVED** [review]

**Pull request merged**